test/helpers.js

```diff
@@ -0,0 +1,27 @@
+// retrieve string that matches regex
+var _contains = function (arr, str) {
+  for (var i = 0, len = arr.length; i < len; i++) {
+    if (arr[i].match(RegExp(str))) {
+      return arr[i].substr(arr[i].indexOf("=") + 1);
+    }
+  }
+  return false;
+};
+
+var endpointWasSet = function () {
```

Comment:
what about a name `getEndpointFromProcessArgs` or something similar and returning `null` in case nothing could be retrieved. I think it will make the intention of the function clearer.